workspace_integration_test.go

```diff
@@ -621,6 +621,32 @@ func TestWorkspacesCreate(t *testing.T) {
 		assert.Equal(t, err, ErrRequiredAgentPoolID)
 	})
 
+	t.Run("when no execution mode is not specified in an organization with local as default execution mode", func(t *testing.T) {
```

Comment:
the double negation made the test summary unclear. 

- [ ]  TODO to update test summary